themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -0,0 +1,9 @@
+@import '~terra-mixins/lib/Mixins';
+
+:global {
+    --terra-dev-site-blockquote-color: #24292e;
+    --terra-dev-site-content-background-color: #fff;
+    --terra-dev-site-icon-link-background-image: inline-svg('<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" class="is-bidi icon icon-link"><path d="M10.5 26.3a16 16 0 0 1 .9-5.3L8 17.6a6.73 6.73 0 0 1-2-4.8A6.41 6.41 0 0 1 8 8a6.76 6.76 0 0 1 9.6 0l9 9a6.88 6.88 0 0 1 0 9.6 6.27 6.27 0 0 1-3.4 1.8 2.65 2.65 0 0 0 .5.6l4.1 4.1a11.36 11.36 0 0 0 3.1-2.3 12.8 12.8 0 0 0 0-18.1l-9-9a12.63 12.63 0 0 0-9-3.7 12.63 12.63 0 0 0-9 3.7A12.1 12.1 0 0 0 0 12.8a12.63 12.63 0 0 0 3.7 9l7 7a21.05 21.05 0 0 1-.2-2.5zm27-4.6a16 16 0 0 1-.9 5.3l3.4 3.4a6.79 6.79 0 1 1-9.6 9.6l-9-9a6.88 6.88 0 0 1 0-9.6 6.27 6.27 0 0 1 3.4-1.8 2.65 2.65 0 0 0-.5-.6l-4-4a11.36 11.36 0 0 0-3.1 2.3 12.8 12.8 0 0 0 0 18.1l9 9a12.73 12.73 0 0 0 18-18l-7-7a8 8 0 0 1 .3 2.3z"/></svg>');
+    --terra-dev-site-layout-menu-panel-panel-background-color: #fff;
+    --terra-dev-site-layout-menu-panel-panel-border-right: #d3d4d5;
```

Comment:
These should be added to [src/navigation/orion-fusion-theme/CollapsingNavigationMenu.module.scss](https://github.com/cerner/terra-dev-site/blob/main/src/navigation/orion-fusion-theme/CollapsingNavigationMenu.module.scss)

Reply:
updated here https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8